packages/sources/src/styling/global/img.scss

```diff
@@ -0,0 +1,3 @@
+img {
+  vertical-align: middle;
```

Comment:
In `_dso-normalize.scss` staat nog een `border: 0`, waar gaat die terugkomen en/of verhuisd worden?

Reply:
Ik kan hem verplaatsen naar `/sources/src/styling/global/img`, maar er staat ook een stukje commentaar boven `Remove border when inside `a` element in IE 8/9/10.` Dus ik vraag me af of dit eigenlijk nog wel nodig is?

Het kan geen kwaad om hem wel mee te nemen lijkt me. Wat vind jij?

Reply:
Laten we maar meenemen, voor hetzelfde geldt leunt er inmiddels andere styling op